### DISCUSSION TITLE
[8/17/23] No Mojo release this week!

### DISCUSSION THREAD

**Discussion by goldiegadde** [modular/modular#504] [Announcements]
Hey everyone 👋 , 

We will **not** be releasing a new Mojo version to the Playground this week! 

Please stay tuned for the next release and in the meantime do check out [An easy introduction to Mojo](https://www.modular.com/blog/an-easy-introduction-to-mojo-for-python-programmers) if you have not already done so!
Thanks, 
Goldie

**Comment by shxrif** [comment]
https://github.com/modularml/mojo/assets/95083469/6e4e1f02-21e7-4781-966d-1b7648e32489

**Comment by shxrif** [comment]
> Hey everyone 👋 , 
> 
> We will **not** be releasing a new Mojo version to the Playground this week! 
> 
> Please stay tuned for the next release and in the meantime do check out [An easy introduction to Mojo](https://www.modular.com/blog/an-easy-introduction-to-mojo-for-python-programmers) if you have not already done so!
> Thanks, 
> Goldie
> 

Hey I am waiting for the AUR release please atleast, whatever u have just upload in AUR we will sort out
